src/WebJobs.Script/Config/ScriptSettingsManager.cs

```diff
@@ -62,19 +58,15 @@ public virtual string AzureWebsiteDefaultSubdomain
         {
             get
             {
-                return _settingsCache.GetOrAdd(nameof(AzureWebsiteDefaultSubdomain), k =>
```

Comment:
Removing all this caching stuff. It was inconsistently used anyways, and wasn't really adding much value at all, while increasing complexity.